producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBOne.java

```diff
@@ -0,0 +1,171 @@
+                                                                                     "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                                                                                     ""),
+                                                                  Lists.newArrayList("USER_REQUEST_ID_B1b",
+                                                                                     "mcolJudgment",
+                                                                                     "Rejected",
+                                                                                     "DUPLD_CUST_REQID",
+                                                                                     "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                                                                                     ""),
+                                                                  Lists.newArrayList("USER_REQUEST_ID_B2",
+                                                                                     "mcolClaim",
+                                                                                     "Rejected",
+                                                                                     "8",
+                                                                                     "First defendant's postcode is not in England or Wales.",
+                                                                                     ""),
+                                                                  Lists.newArrayList("USER_REQUEST_ID_B3",
+                                                                                     "mcolJudgment",
+                                                                                     "Initially Accepted",
+                                                                                     "", "",
+                                                                                     """
+                                                                                          <![CDATA[
+                                                                                                                   <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
+                                                                                                                      <mresp:firstPaymentDate>2014-02-14</mresp:firstPaymentDate>
+                                                                                                                   </ind:mcolResponseDetail>
+                                                                                                                 ]]>
```

Comment:
Appears to be lots of extra indentation inside the text block compared to the original which was like this:
```
<![CDATA[
  <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
  <mresp:firstPaymentDate>2014-02-14</mresp:firstPaymentDate>
  </ind:mcolResponseDetail>
]]>
```

Reducing the indentation to how it was will improve the readability.

Reply:
done this indentation